atd-vzv/src/views/summary/FatalitiesByMode.js

```diff
@@ -13,7 +14,6 @@ const FatalitiesByMode = () => {
     { label: "Motorcycle", flag: "motorcycle_fl", color: "#de2d26" },
```

Comment:
I know you didn't add these here in this PR but we should probably extract those color hex IDs into a shared place.